lib/AST/ASTPrinter.cpp

```diff
@@ -2546,8 +2546,11 @@ void PrintAST::visitFuncDecl(FuncDecl *decl) {
         [&]{ // Name
           if (!decl->hasName())
             Printer << "<anonymous>";
-          else
+          else {
```

Comment:
Nitpick: if the `else` has braces, can you add them to the `if` as well?

Reply:
I was using the "only add braces if there are multiple statements" rule that seems to be used in most code, but I'm 👍 on changing it.

Reply:
Heh. That is the general rule, but at least for my code I've kept it consistent across multiple branches of an `if` statement, on the grounds that the intent is to only omit the braces when it's easy to read as a single unit.